packages/localize/src/rollup.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as ts from 'typescript';
+import {
+  TransformLitLocalizer,
+  TransformOutputConfig,
+} from './modes/transform.js';
+import {readConfigFileAndWriteSchema, Config} from './config';
+
+/**
+ * Return an array of locales with associated TypeScript transformer factories
+ * suitable for passing to the `transformers.before` [0] option of
+ * `@rollup/plugin-typescript`.
+ *
+ * Example:
+ *
+ *   const locales = localeTransformers()
+ *   export default locales.map(({locale, localeTransformer}) => ({
+ *     input: `src/index.ts`,
+ *     plugins: [
+ *       typescript({
+ *         transformers: {
+ *           before: [localeTransformer],
+ *         },
+ *       }),
+ *       resolve(),
+ *     ],
+ *     output: {
+ *       file: `bundled/${locale}/index.js`,
+ *       format: 'es',
+ *     },
+ *   }));
+ *
+ * [0] https://github.com/rollup/plugins/tree/master/packages/typescript/#transformers
+ */
+export const localeTransformers = (
+  configPath = './lit-localize.json'
+): Array<{
```

Comment:
Recommend giving this type a name, both so this code is more readable, and so that user code can more easily name it.

Reply:
Done.